doc/test-framework/scorecard.md

```diff
@@ -325,11 +325,157 @@ Example of a valid JSON output:
 
 **NOTE:** The `ScorecardOutput.Log` field is only intended to be used to log the scorecard's output and the scorecard will ignore that field if a plugin provides it.
 
-## Running the scorecard with a deployed CSV
-
-The scorecard can be run using only a [Cluster Service Version (CSV)][olm-csv], providing a way to test cluster-ready and non-SDK operators.
+## Running the scorecard with an OLM-managed operator
+
+The scorecard can be run using a [Cluster Service Version (CSV)][olm-csv], providing a way to test cluster-ready and non-SDK operators.
+
+Running with a CSV alone requires both the `--csv-path=<CSV manifest path>` and `--olm-deployed` flags to be set. The scorecard assumes your CSV and relevant CRD's have been deployed onto the cluster using OLM when using `--olm-deployed`.
+
+The scorecard requires a proxy container in the operator's `Deployment` pod to read operator logs. A few modifications to your CSV and creation of one extra object are required to run the proxy _before_ deploying your operator with OLM:
+
+1. Create a proxy server secret containing a local Kubeconfig:
```

Comment:
It shows that required the data from the local Kubeconfig when I understand the following steps are related to the cluster where the OLM files will bee deployed and tested/checked with the score 
```suggestion
1. Create a proxy server secret for your cluster:
```